HousingRepairsSchedulingApi.Tests/ServicesTests/Drs/DrsServiceTests.cs

```diff
@@ -305,6 +309,8 @@ public async void GivenScheduleBookingResponse_WhenSchedulingABooking_ThenDrsSoa
 
             // Assert
             soapMock.Verify(schedulingBookingExpression);
+            Assert.True(request.scheduleBooking1.theBooking.assignedStartSpecified);
+
```

Comment:
Could you also test assignedEndSpecified?

Reply:
Done